bigquery/google/cloud/bigquery/client.py

```diff
@@ -603,7 +603,7 @@ def get_job(
         Returns:
             Union[google.cloud.bigquery.job.LoadJob, \
                   google.cloud.bigquery.job.CopyJob, \
-                  google.cloud.bigquery.job.ExtractJob \
+                  google.cloud.bigquery.job.ExtractJob, \
```

Comment:
These fixes are uncontroversial, as well as unrelated to the main thrust of this PR.  Maybe they should move to their own PR?

Reply:
Good idea. Mailed https://github.com/GoogleCloudPlatform/google-cloud-python/pull/5342